Assets/Scripts/Actors/ActorComponents/PlayerActorInventory.cs

```diff
@@ -116,7 +120,21 @@ void SpawnBuddy()
 		BuddyStats newBuddy = ( Instantiate( buddyItemData.buddyPrefab,
 		                                     transform.position + transform.forward * 3.0f,
 		                                     Quaternion.identity ) as GameObject ).GetComponent<BuddyStats>();
-		newBuddy.SetGod( GetComponent<GodTag>() );
+		newBuddy.owner = GetComponent<GodTag>();
+		newBuddy.statType = buddyItemData.stat;
+
+		MeshRenderer[] childRenderers = newBuddy.gameObject.GetComponentsInChildren<MeshRenderer>();
+
+		foreach ( MeshRenderer meshRenderer in childRenderers )
+		{
+			if ( meshRenderer.gameObject.name == "Body" )
+			{
+				// buddyItemData.prefab.GetComponentInChildren<MeshRenderer>() was not working
+				MeshRenderer render = buddyItemData.prefab.transform.FindChild( "Sphere" ).gameObject.GetComponent<MeshRenderer>();
+				meshRenderer.material = render.material;
+			}
```

Comment:
Is this changing the buddies color to match the resource color? We should store a reference or make a getter in buddy item data to it's material.